src/main/java/com/ericsson/sm/CarApp/dto/ClientRequestDto.java

```diff
@@ -15,5 +18,7 @@ public class ClientRequestDto {
     private String zipCode;
     private String country;
 
+    private List<Car> cars;
```

Comment:
Zašto u ClientRequestDto imamo listu auta? Nigdje ne piše da se kod kreiranja klijenta moraju kreirati i auti. Moram i Stjepanu reći da izbaci taj atribut